lib/web/css/source/lib/variables/_typography.less

```
@@ -39,11 +39,15 @@
 @font-size__xs: floor(.75 * @font-size__base); // 11px
 
 //  Weights
+@font-weight__hairline: 100;
+@font-weight__thin: 200;
```

from the table: 

Value | Common weight name
-- | --
200 | Extra Light (Ultra Light)



```suggestion
@font-weight__extralight: 200;
```